mkdocs.yml

```diff
@@ -79,6 +79,7 @@ nav:
             - Taxonomy events: api/event_reference/taxonomy_events.md
             - Role events: api/event_reference/role_events.md
             - User events: api/event_reference/user_events.md
+            - Segmentation events: api/event_reference/segmentation_events.md
```

Comment:
Out of scope, but have you considered rearranging the order of events in the TOC to be alphabetical, with "Reference" at the top?

Reply:
I actually did but decided not to. I can do it in a separate PRs, one for each branch